Mailgun_Subscriptions/Confirmation.php

```diff
@@ -64,7 +64,7 @@ public function save() {
 				'post_type' => self::POST_TYPE,
 				'post_status' => 'publish',
 				'post_author' => 0,
-				'post_title' => $this->id,
+				'post_title' => sprintf( '%s (%s)', get_post_type_object( static::POST_TYPE )->labels->singular_name, $this->id ),
```

Comment:
This change was to provide context when viewing the WordPress database `posts` table.